CHANGELOG.md

```diff
@@ -19,6 +19,11 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### BREAKING
 - Rename `DropdownLabel` to `DropdownSelectedItem` and extract styles @layershifter ([#725](https://github.com/stardust-ui/react/pull/725))
+- Changes in `Dropdown` @layershifter ([#729](https://github.com/stardust-ui/react/pull/729))
+  - rename `noResultsMessage` prop to `messageNoResults`
+  - `messageNoResults` has default value by default 
+  - rename `toggleButton` prop to `indicator`
```

Comment:
could you, please, mention the libraries you've compared against (other than SUIR) - those that server as a reason for this change?

Reply:
SUIR calls this prop `icon`, but in the current implementation it's an unicode character.
---
https://react-select.com/props
And we have now the `Indicator` component.

From other side it's that `toggleButton` can behave like a loader.

Reply:
We decided to go with `toggleIndicator` :ship